renderer/pyscript-offline/public/README.md

```diff
@@ -0,0 +1,19 @@
+# UNLWS Renderer
+A tool to specify UNLWS texts in code and display them.
+
+## Setup
+
+1. Clone the repo
+2. Open a terminal in `...\unlws\renderer\pyscript-offline\public`
```

Comment:
Should be forward slashes in Unix systems. Are you on Windows?

Reply:
Fixed it. Yes I'm on Windows